astropy/nddata/ccddata.py

```diff
@@ -535,7 +556,7 @@ def _generate_wcs_and_update_header(hdr):
 
 def fits_ccddata_reader(filename, hdu=0, unit=None, hdu_uncertainty='UNCERT',
                         hdu_mask='MASK', hdu_flags=None,
-                        key_uncertainty_type='UTYPE', **kwd):
+                        key_uncertainty_type='UTYPE', hdu_psf="PSFIMAGE", **kwd):
```

Comment:
```suggestion
                        key_uncertainty_type='UTYPE', hdu_psf='PSFIMAGE', **kwd):
```
Again, see comment above (for these cases I have no idea if the quote question becomes all moot once this is all subjected to `black`, but until then I'd keep the style consistent).